src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_30">
+      <div class="calendar__day calendar__day--first-day_sun"></div>
```

Comment:
According to the task, `first-day` it's a `calendar` modifier, not `calendar__day`.
And make sure to follow BEM naming convention for complex modifiers: block-name--modifier-name--modifier-value (checklist)

```suggestion
    <div class="calendar calendar--month-length--30 calendar--first-day--sun">
      <div class="calendar__day "></div>
```